src/helpers.ts

```diff
@@ -119,53 +119,57 @@ export function makeDirIfMissing(dir: string) {
 }
 
 const sourcesInfo: Record<number, string> = {};
-export interface Categories {
-  sources: Record<
-    string, // a sourceTag. i.e. "adventures" or "deadorbit" or "zavala" or "crucible"
-    {
-      /**
-       * list of strings. if a sourceString contains one of these,
-       * it probably refers to this sourceTag
-       */
-      includes: string[];
-      /**
-       * list of strings. if a sourceString contains one of these,
-       * it doesn't refer to this sourceTag
-       */
-      excludes?: string[];
-      /** list of english item names or inventoryItem hashes */
-      items?: (string | number)[];
-      /** duplicate this category into another sourceTag */
-      alias?: string[];
-      /**
-       * presentationNodes can contain a set of items (Collections).
-       * we'll find presentation nodes by name or hash,
-       * and include their children in this source
-       */
-      presentationNodes?: (string | number)[];
-      searchString?: string[];
-      originTrait?: string[];
-      excludedItems?: string[];
-      supplements?: string[];
-    }
-  >;
-  /** give better descriptions to match against etc, easy add IGNORE */
-  renameSourceStrings: string[][];
+
+interface Categories {
+  /** a sourceTag. i.e. "adventures" or "deadorbit" or "zavala" or "crucible" */
+  sourceName: string;
+  /**
+   * list of strings. if a sourceString contains one of these,
+   * it probably refers to this sourceTag
+   */
+  desc?: string[];
+  /**
+   * list of strings. if a sourceString contains one of these,
+   * it doesn't refer to this sourceTag
+   */
+  excludes?: string[];
+  /** list of english item names or inventoryItem hashes */
+  items?: (string | number)[];
+  /**
+   * presentationNodes can contain a set of items (Collections).
+   * we'll find presentation node by name or hash */
+  presentationNodes?: (string | number)[];
+  /**
+   * duplicate this category into another sourceTag
+   */
+  alias?: string[];
+  /**
+   *  this sourceTag and all its itemsHashes and sourceHashes
+   *  should be added to this additional sourceTag
+   */
+  supplements?: string[];
+  /** originTrait english name that matches this sourceTag */
+  originTrait?: string[];
+  /**
+   * english name of items containing the appropriate originTrait
+   * but they should not be considered part of this sourceTag
+   */
+  excludedItems?: string[];
```

Comment:
not sure this still needs to be here, just not sure how to get the info here from category-config.ts as an interface